app/components/ui-kit/font.js

```diff
@@ -0,0 +1,83 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { none } from '@ember/object/computed';
+import { checkDictionary } from 'travis/utils/ui-kit/assertions';
+import spacingMixin from 'travis/mixins/ui-kit/spacing';
+
+const FAMILIES = {
+  SANS: 'sans',
+  SERIF: 'serif',
+  MONO: 'mono',
+};
+
+const SIZES = {
+  XS: 'xs',
+  SM: 'sm',
+  BASE: 'base',
+  LG: 'lg',
+  XL: 'xl',
+  XL2: '2xl',
+  XL3: '3xl',
+  XL4: '4xl',
+  XL5: '5xl',
+  XL6: '6xl',
+};
+
+const WEIGHTS = {
+  LIGHT: 'light',
+  NORMAL: 'normal',
+  SEMIBOLD: 'semibold',
+  BOLD: 'bold',
+};
+
+const TRANSFORMS = {
+  UPPERCASE: 'uppercase',
+  LOWERCASE: 'lowercase',
+  CAPITALIZE: 'capitalize',
+  NORMAL: 'normal-case',
+};
+
+// Component definition
+export default Component.extend(spacingMixin, {
+  tagName: '',
+
+  // Public interface
+  tag: 'span',
+
+  color: null,
+  family: null,
+  size: null,
+  transform: null,
+  weight: null,
```

Comment:
For the general components that are going to be used a lot, like `Box` and `Text`, I was thinking it would be nice to apply as few classes by default  as possible. But I can see the value in either approach. So if you feel like that's the way to go, I can definitely put some default values on those props.

Reply:
I think it would make sense to set some sensible defaults to all components, so that they render properly with no attributes passed at all. E.g. just `<Button>Title</Button>` should render some default button, which then could be adjusted by setting certain attributes. For `Text` component, I'd suggest to make it just as a regular paragraph (`<p>`) by default. What do you think?

Reply:
Sure, that seems sensible. I added some defaults to the component.